pkg/package/toml.go

```diff
@@ -82,7 +82,7 @@ func (dep *Dependencies) MarshalTOML() string {
 const DEP_PATTERN = "%s = %s"
 
 func (dep *Dependency) MarshalTOML() string {
-	source := dep.Source.MarshalTOML()
+	source := dep.Source.MarshalTOML(dep.Name)
```

Comment:
Do not go this way, because the purpose of adding the `package` field is to support renaming dependency, you need to add a field on the `Dependency` structure to hold the package name and use this field when serializing to kcl.mod. Instead of simply using `dep.Name`, the value of `package` will not be the same as `dep.Name` when the rename function is implemented later.